src/DIRAC/FrameworkSystem/Agent/ComponentSupervisionAgent.py

```diff
@@ -565,7 +569,7 @@ def _checkServiceURL(self, serviceName, options):
     module = options['Module']
     self.log.info('Checking URLs for %s/%s' % (system, module))
     urlsConfigPath = PathFinder.getSystemURLSection(
-        serviceName=os.path.join(system, module),
+        serviceName=Path.cfgPath(system, module),
```

Comment:
why ? `cfgPath` is nothing more than `os.path.join`

Reply:
Well, it's not exactly the same, `cfgPath` is `os.path.normpath(os.path.join(*(str(k) for k in args)))`

Reply:
I believe the output of `join` already is normalized, except that it does not "traverse" the path to remove `..` or `.` But we never have that for the CS path

Reply:
It can go wrong in a few ways:

```python
In [17]: os.path.join("system//", "2")
Out[17]: 'system//2'

In [18]: os.path.join("system//", "//2")
Out[18]: '//2'

In [19]: os.path.join("system//", "2/")
Out[19]: 'system//2/'

In [20]: os.path.join("system//", "2//")
Out[20]: 'system//2//'
```

While really we want:
```python
In [21]: Path.cfgPath("system//", 2)
Out[21]: 'system/2'
```

Reply:
This one gives the same for `cfgPath`:

```python
In [18]: os.path.join("system//", "//2")
Out[18]: '//2'
```

But okay I see your point. Basically, one has to avoid giving stupid inputs... :-)